src/main/java/com/teragrep/cfe_41/Main.java

```diff
@@ -0,0 +1,90 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://github.com/teragrep/teragrep/blob/main/LICENSE>.
+ *
+ *
+ * Additional permission under GNU Affero General Public License version 3
+ * section 7
+ *
+ * If you modify this Program, or any covered work, by linking or combining it
+ * with other code, such other code is not for that reason alone subject to any
+ * of the requirements of the GNU Affero GPL version 3 as long as this Program
+ * is the same Program as licensed from Suomen Kanuuna Oy without any additional
+ * modifications.
+ *
+ * Supplemented terms under GNU Affero General Public License version 3
+ * section 7
+ *
+ * Origin of the software must be attributed to Suomen Kanuuna Oy. Any modified
+ * versions must be marked as "Modified version of" The Program.
+ *
+ * Names of the licensors and authors may not be used for publicity purposes.
+ *
+ * No rights are granted for use of trade names, trademarks, or service marks
+ * which are in The Program if any.
+ *
+ * Licensee must indemnify licensors and authors for any liability that these
+ * contractual assumptions impose on licensors and authors.
+ *
+ * To the extent this program is licensed as part of the Commercial versions of
+ * Teragrep, the applicable Commercial License may apply to this file if you as
+ * a licensee so wish it.
+ */
+package com.teragrep.cfe_41;
+
+import com.teragrep.cfe_41.flow.FlowRequest;
+import com.teragrep.cfe_41.flow.FlowResponse;
+import com.teragrep.cfe_41.flow.PartialFlowResponse;
+import com.teragrep.cnf_01.ArgsConfiguration;
+import com.teragrep.cnf_01.Configuration;
+import com.teragrep.cnf_01.ConfigurationException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.util.List;
+import java.util.ArrayList;
+
+public class Main {
+
+    public static void main(final String[] args) throws Exception {
+        final Logger logger = LogManager.getLogger(Main.class);
+        // Creates new ApiConfig from commandline args
+        final Configuration configuration = new ArgsConfiguration(args);
+        Map<String, String> configMap = new HashMap<>();
+        try {
+            logger.debug("Loaded configuration <{}>", configuration.asMap());
+            configMap = configuration.asMap();
+        }
+        catch (ConfigurationException e) {
+            logger.error("Error loading configuration <{}>", e.getMessage());
+        }
+
+        final ApiConfig apiConfig = new ApiConfig(configMap);
+
+        final FlowRequest flowRequest = new FlowRequest(apiConfig);
```

Comment:
this is test case content, please remove from main and run it in a test case annotated with a manualRun equals yes property if needed so you can 
mvn clean test -Dtest=testName#testMethodName -DmanualRun=true

see https://github.com/teragrep/rlp_03/blob/cd0cd2077157087433219b83df7c2eafd75a0b93/src/test/java/com/teragrep/rlp_03/ManualTest.java#L75-L79